internal/executor/endpoint/endpoint.go

```diff
@@ -0,0 +1,44 @@
+package endpoint
+
+type Endpoint interface {
+	Container() string
+	Direct() string
```

Comment:
Could you please add comments explaining the difference between the endpoints so there is no need to check out the usage in order to understand it.

Reply:
Explained in https://github.com/cirruslabs/cirrus-cli/pull/495/commits/cda040c0edd2a41eb6742eb0e826478e551241ad.